File: 1134-uri.c
#include <stdio.h>
#include <stdlib.h>
 
int main() {
    int alcool=0,gasolina=0,diesel=0,i,combustivel=0;
    while (combustivel!=4){
        scanf("%d",&combustivel);
        if(combustivel==1){
            alcool=alcool+1;
        }if(combustivel==2){
            gasolina =gasolina+1;
        }if (combustivel==3){
            diesel=diesel+1;
        }if (combustivel<1 ||combustivel>4){
            combustivel=0;
        }
    }printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n",alcool);
    printf("Gasolina: %d\n",gasolina);
    printf("Diesel: %d\n",diesel);
    
    return 0;
}

File: 1134-uri.py
combustivel = 0
contador1=0
contador2=0
contador3=0
while combustivel != 4 :
    combustivel=int(input())
    if combustivel == 1:
        contador1 +=1
    if combustivel ==2:
        contador2 +=1
    if combustivel ==3:
        contador3 +=1
print("MUITO OBRIGADO")
print("Alcool:",contador1)
print("Gasolina:",contador2)
print("Diesel:",contador3)
